src/test/java/net/snowflake/client/TestUtil.java

```diff
@@ -75,7 +75,11 @@ public interface TestRunInterface {
    */
   public static String systemGetEnv(String env) {
     try {
-      return System.getenv(env);
+      String value = System.getenv(env);
```

Comment:
can we remove this method and and use SnowflakeUtil?

Reply:
I will try and let you know if is possible. Thank you!!

Reply:
I tried to remove the TestUlt.systemGetEnv, but it has issues when we try to validate the Old JDBC driver because that version is not available; that's why it was duplicated. https://github.com/snowflakedb/snowflake-jdbc/actions/runs/9352721781/job/25741532405?pr=1776

Reply:
@sfc-gh-dprzybysz  Can I resolve this comment?

Reply:
It's already resolved